lte/gateway/c/session_manager/test/ProtobufCreators.cpp

```diff
@@ -110,6 +111,20 @@ void create_rule_record(
   rule_record->set_ue_ipv4(ip);
 }
 
+void create_rule_record(
+    const std::string& imsi, const std::string& ip, const std::string& rule_id,
```

Comment:
why are you using IP here. Didn't we move to TEID?

Reply:
that seems to be delayed so this was actually a good decision to use IP 😛